test/utils/mockutils/mockserver/handlers.go

```diff
@@ -0,0 +1,26 @@
+package mockserver
+
+import "net/http"
+
+// ContentJSON is a setup handler, which configures server to use JSON.
+func ContentJSON() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+	}
+}
+
+// Response is used to configure server response with HTTP status and body data.
+// Data is optional.
+func Response(status int, data ...[]byte) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(status)
+
+		if len(data) == 1 {
```

Comment:
What happens if len > 1?

Reply:
I will add a panic. This is a "hack" to allow optional field. Not all mocks respond with data. Adding new method would not make it look nice, although proper style.

The reason for panic is to show that whoever created a test did a mistake.